tests/message_properties/definitions/test_genome_size.py

```diff
@@ -0,0 +1,6 @@
+from tol_lab_share.message_properties.definitions.genome_size import GenomeSize
+from helpers import check_validates_string
+
+
+def test_genome_size_is_string():
```

Comment:
Should genome size be a string? What sort of values would we expect?

Reply:
I couldnt see any examples of the value; for similar fields (concentration, volume) we are also using string that are parsed as float when reading the message. In this case I was leaving it as string until that is more clear.